modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CppRestClientCodegen.java

```diff
@@ -391,7 +389,8 @@ public String escapeQuotationMark(String input) {
 
     @Override
     public String escapeUnsafeCharacters(String input) {
-        return input.replace("*/", "*_/").replace("/*", "/_*");
+        // escape nothing for CppRest
```

Comment:
We need this to prevent code injection and the latest master should not escape `*/*`